util/cron/common-native-gpu-perf.bash

```diff
@@ -5,4 +5,4 @@
 # set this before sourcing common-perf
 export CHPL_TEST_PERF_SUBDIR='gpu'
 
-nightly_args="${nightly_args} -performance -perflabel gpu- -numtrials 5"
+nightly_args="${nightly_args} -no-buildcheck -performance -perflabel gpu- -numtrials 5"
```

Comment:
I'm OK with this as this is not a correctness config. But also curious whether there is any deep reason beyond that.

Reply:
I don't have a deep reason beyond the fact that many of the other perf configs do this